content/handling-objection/_index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Chapter 2: Handling Objection"
+weight: 102
+---
+
+## What Happens After No
+***
+We all will have the unfortunate privilege of interacting with someone that is difficult to work with in our personal or professional lives.. We can check our actions and behaviors to ensure we are not making the same mistake as others may have with the same customer or person (managers, co-workers, spouse etc.). Acknowledge and address objections rather than ignoring or dismissing them. Grabbing the objections off the table when they are added allow us to gain better insight and understanding of the situation. A “No” in business rarely means that all the facts have been considered and all options have been exhausted. It’s usually our way of protecting ourselves from unknown outcomes from in the agreement. When we offer a solution to a speaker/customer, and they refuse or reject our offer, we can ask open ended questions to see if there is something we missed. For example, A customer calls in and angry and only wants to talk to a manager/supervisor. We ask if there is anything we can do to help and they say No. What do you do? Take a moment and think about what you will do. Most of us would connect the customer to a manager and move on with our day. However, we can take a few steps prior to the transfer if we ask the right questions.
```

Comment:
```suggestion
We all will have the unfortunate privilege of interacting with someone that is difficult to work with in our personal or professional lives. We can check our actions and behaviors to ensure we are not making the same mistake as others may have with the same customer or person (managers, co-workers, spouse etc.). Acknowledge and address objections rather than ignoring or dismissing them. Grabbing the objections off the table when they are added allow us to gain better insight and understanding of the situation. A “No” in business rarely means that all the facts have been considered and all options have been exhausted. It’s usually our way of protecting ourselves from unknown outcomes from in the agreement. When we offer a solution to a speaker/customer, and they refuse or reject our offer, we can ask open ended questions to see if there is something we missed. For example, A customer calls in and angry and only wants to talk to a manager/supervisor. We ask if there is anything we can do to help and they say No. What do you do? Take a moment and think about what you will do. Most of us would connect the customer to a manager and move on with our day. However, we can take a few steps prior to the transfer if we ask the right questions.
```